web-client/README.md

```diff
@@ -1,46 +0,0 @@
-# Getting Started with Create React App
```

Comment:
It would be better to have a README file for the project, but update it.

Reply:
Done. With the comands for run and build the app - npm run dev, npm run build